drivers/bus/isapnp/isapnp.c

```diff
@@ -957,9 +785,12 @@ IsaPnpCreateLogicalDeviceResources(
                                   LogDev->MemRange32[i].CurrentBase,
                                   LogDev->MemRange32[i].CurrentLength,
                                   NULL,
-                                  &Information,
-                                  NULL))
+                                  &Information))
         {
+            DPRINT1("MEM entry #%lu %lx %lx not found\n",
```

Comment:
"MEM32 entry" maybe, to distinguish in the debug log?

Reply:
Ok